integration-tests/contracts/contract_deployer.go

```diff
@@ -877,6 +1007,21 @@ func (e *EthereumContractDeployer) LoadKeeperRegistry(address common.Address, re
 			client:      e.client,
 			registry2_0: instance.(*keeper_registry_wrapper2_0.KeeperRegistry),
 		}, err
+	case eth_contracts.RegistryVersion_2_1:
+		instance, err := e.client.LoadContract("KeeperRegistry", address, func(
+			address common.Address,
+			backend bind.ContractBackend,
+		) (interface{}, error) {
+			return registry21.NewKeeperRegistry(address, backend)
```

Comment:
I think here we should be using the master interface, similar to that we do in [chaincli](https://github.com/smartcontractkit/chainlink/blob/f0ae2c451ed8572471dd9d41b2a14289d43aa8c0/core/scripts/chaincli/handler/keeper.go#L310)